README.md

```diff
@@ -1,36 +1,42 @@
 # [Squoosh]!
 
-[Squoosh] is an image compression web app that allows you to dive into the advanced options provided
-by various image compressors.
+[Squoosh] is an image compression web app that provides lossless image quality with a significant reduction to file size.
```

Comment:
“provides lossless image quality” doesn’t sit well with me. We do mostly lossy compression. 
Maybe we can find a phrasing around “lets you explore image formats” or something? Also Squoosh is very much targeted at web developers, so you can lean into that if you think it makes sense.